contracts/VToken.sol

```diff
@@ -581,7 +581,7 @@ contract VToken is Ownable2StepUpgradeable, VTokenInterface, ExponentialNoError,
 
         /*
          *  We call `_doTransferIn` for the minter and the mintAmount.
-         *  Note: The vToken must handle variations between ERC-20 and ETH underlying.
+         *  Note: The vToken must handle variations of ERC-20 underlying.
```

Comment:
What would this sentence mean? I think we can remove the sentence

Reply:
Agree. I have removed the sentence.